WorkloadManagementSystem/Service/SandboxStoreHandler.py

```diff
@@ -244,15 +244,14 @@ def __networkToFile( self, fileHelper, destFileName = False ):
     except:
       pass
     try:
-      fd = open( destFileName, "wb" )
-    except Exception, e:
+      with open( destFileName, "wb" ) as fd:
+        result = fileHelper.networkToDataSink( fd, maxFileSize = self.__maxUploadBytes )
+    except Exception as e:
       return S_ERROR( "Cannot open to write destination file %s" % destFileName )
-    result = fileHelper.networkToDataSink( fd, maxFileSize = self.__maxUploadBytes )
     if not result[ 'OK' ]:
       return result
     if tfd:
```

Comment:
not that I can see in the code as the file is once more open as "fd"... This last opening should be avoided and tfd used instead as the documentation says the file is open as binary